### DISCUSSION TITLE
GCN - How to predict new data?

### DISCUSSION THREAD

**Discussion by Matheusutino** [pyg-team/pytorch_geometric#5592] [General]
Hello, 

I was studying pytorch GCN, in this case the notebook from this [link](https://colab.research.google.com/drive/14OvFnAXggxB8vM4e8vSURUp1TaKnovzX?usp=sharing#scrollTo=pcr9joFQ6Mri) for cora dotaset.

I want to know how to predict new nodes, because after training, validating and testing how would I test new data after that?

Thanks in advance

**Comment by rusty1s** [comment]
For testing on new nodes, you need to add these nodes to your graph/`data` object first. After that, you can simply run your trained model on it just as before.
